### PR TITLE
Add a new updates notifier using Sparkle

### PR DESCRIPTION
**Pull request by ylatuya**

### CODE REVIEW

File: VAS.Services/AppUpdater/SimplifiedAppUpdater.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+		}
+
+		async Task<string> FetchNewVersion (string url)
+		{
+			string appcastStr = null;
+			try {
+				var wb = new WebClient ();
+				appcastStr = await wb.DownloadStringTaskAsync (url);
+			} catch (Exception ex) {
+				Log.WarningFormat ($"UpdatesNotifier: Error downloading version file from {url}");
```
Comment (vmartos) [inline VAS.Services/AppUpdater/SimplifiedAppUpdater.cs]: Log.Warning

File: VAS.Services/AppUpdater/SimplifiedAppUpdater.cs [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+				Log.Exception (ex);
+				return appcastStr;
+			}
+			Log.InformationFormat ($"UpdatesNotifier: Downloaded latest version from {url}");
```
Comment (vmartos) [inline VAS.Services/AppUpdater/SimplifiedAppUpdater.cs]: Log.Information

File: VAS.Services/AppUpdater/SparkleOSX.cs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+				throw new Exception ($"{infoPlist} not found");
+			}
+			string bundlePath = Path.GetFullPath (Path.Combine (infoPlist, "..", ".."));
+			Log.Information ($"Found bundle at path {bundlePath}");
+			return bundlePath;
+		}
+
+		void LogError (string msg)
+		{
+			Log.Error ($"Updater will not start: {msg}");
```
Comment (vmartos) [inline VAS.Services/AppUpdater/SparkleOSX.cs]: Redundant Method, is only used once in the class.

File: VAS.Services/AppUpdater/SimplifiedAppUpdater.cs (hunk 1 above)
Comment (ylatuya) [inline VAS.Services/AppUpdater/SimplifiedAppUpdater.cs]: right

File: VAS.Services/AppUpdater/SparkleOSX.cs (hunk 3 above)
Comment (ylatuya) [inline VAS.Services/AppUpdater/SparkleOSX.cs]: right, it used to be used more than once but not anymore

**Comment by ylatuya** [comment]
all fixed now

**Pull request merged**